modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -450,10 +448,10 @@ paths:
                     description:
                         default response
             tags: ["UserGroup"]
-        post:
+        get:
```

Comment:
I think that this get method is not necessary.

To retrieve a single UserGroup we could use the proper API. (user admin)

Reply:
The autogenerated tests for PUT require a GET, so we have to keep it.